### PR TITLE
(Builder) Update Application Landing page for when applications have not opened yet 

### LINKED ISSUE #1761: Builder: update Application Landing page for when applications have not opened yet
**User Stories**
As a project owner 
I want to know if the application period for a round has started yet 
So that I know when I can start applying 

**Acceptance Criteria**
GIVEN that applications have not started
WHEN I look at the page 
THEN I see information about applications starting on X date
AND a disabled Apply CTA

**Product & Design links**
* [Figma](https://www.figma.com/file/obBzCCL8KwoTlVBb5g3ccN/Builder-S17?type=design&node-id=4891-39873&t=AXp18ripODoi3NBL-4) 
* Try: https://staging.grantshub.gitcoin.co/#/chains/5/rounds/0xd8b77930ff589a27a7c15d825eb04860669da56b

### PR DESCRIPTION
**Pull request by hmrtn**
Fixes #1761

### CODE REVIEW

**Review by boudra — APPROVED** [review]

**Comment by willsputra** [comment]
@hmrtn just tested this - I got an Invalid Date for a round that starts in the future (https://bafybeic7w6aif2pvxvnl7b4hf4ixbvrbge4al5tabqmpil6ueyumhdceae.on.fleek.co/#/chains/5/rounds/0x979673178bad824a5964c0ff62e9235b38bfc4c4)

image

**Comment by michellema1208** [comment]
@hmrtn it looks like the build for Builder is queued, so am not able to test atm!

**Review by gravityblast — APPROVED** [review]

File: packages/builder/src/components/rounds/Show.tsx [hunk 1]
```diff
@@ -17,6 +17,100 @@ import Button, { ButtonVariants } from "../base/Button";
 import ErrorModal from "../base/ErrorModal";
 import LoadingSpinner from "../base/LoadingSpinner";
 import SwitchNetworkModal from "../base/SwitchNetworkModal";
+import { GrantsMetadataState } from "../../reducers/grantsMetadata";
+
+interface ApplyButtonProps {
+  roundState: any;
```
Comment (bhargavaparoksham) [inline packages/builder/src/components/rounds/Show.tsx:23]: @boudra can we update this from 'any' to the actual type?

**Review by bhargavaparoksham — CHANGES_REQUESTED** [review]

**Comment by bhargavaparoksham** [comment]
@boudra can we add / update the tests for this as well? [Link](https://github.com/gitcoinco/grants-stack/blob/dd85f266dc68f2d7fd9d1fa80091b24894e4447f/packages/builder/src/__tests__/components/rounds/Show.test.tsx)

File: packages/builder/src/components/rounds/Show.tsx (hunk 1 above)
Comment (boudra) [reply packages/builder/src/components/rounds/Show.tsx]: resolved!

**Review by gravityblast — DISMISSED** [review]

**Review by DanieleSalatti — APPROVED** [review]

**Review by bhargavaparoksham — APPROVED** [review]

**Review by melissa-neira — APPROVED** [review]

**Pull request merged**